public/locales/en.json

```diff
@@ -252,7 +258,8 @@
             "deleteNote": {
                 "title": "Delete note",
                 "question": "Do you really want to delete this note?",
```

Comment:
same here

Reply:
Done, even if I'm a bit unsure about that, because the same modal is used in the editor's dropdown menu and from the editor the saying 'this note' is more clearly.